### PR TITLE
cast occurred_at as timestamp

### PR DESCRIPTION
**Pull request by fivetran-jamie**
addressing https://github.com/fivetran/dbt_mixpanel/issues/11

### CODE REVIEW

**Comment by fivetran-jamie** [comment]
this is gonna fail on circle because we have not yet merged the [PR](https://github.com/fivetran/dbt_mixpanel/pull/9) that implements circle testing

File: models/staging/stg_mixpanel__event.sql [hunk 1]
```diff
@@ -12,6 +12,7 @@ fields as (
     select
         cast( {{ dbt_utils.date_trunc('day', 'time') }} as date) as date_day,
         lower(name) as event_type,
+        cast(time as {{ dbt_utils.type_timestamp() }} )as occurred_at,
```
Comment (CraigWilson-ZOE) [inline models/staging/stg_mixpanel__event.sql:15]: Missing space after the end `)`, the compiled code currently looks like:
```
cast(time as 
    timestamp
 )as occurred_at,
```

**Comment by CraigWilson-ZOE** [comment]
I have tested this with my dbtcloud project and the BigQuery error no longer appears, so this fix appears to be working as expected, thanks.

File: models/staging/stg_mixpanel__event.sql (hunk 1 above)
Comment (fivetran-jamie) [reply models/staging/stg_mixpanel__event.sql]: ah good catch -- fixed!

**Review by fivetran-kristin — APPROVED** [review]
LGTM

**Pull request merged**